app/services/find_bulk_recommend_trainees.rb

```diff
@@ -3,18 +3,14 @@
 class FindBulkRecommendTrainees
   include ServicePattern
 
-  # rubocop:disable Style/TrailingCommaInArguments
   def call
     itt_end_date_range = [(Time.zone.today - 6.months).iso8601, (Time.zone.today + 6.months).iso8601]
 
-    Trainee
-      .where(state: :trn_received)
-      .where(
-        <<~SQL
-          '#{itt_end_date_range}'::daterange @> trainees.itt_end_date OR
-          trainees.itt_end_date IS NULL
-        SQL
-      ).order(last_name: :asc)
```

Comment:
I removed the ordering from this service because in all but one case we use it to count the number of possible bulk recommend trainees. It's only in the reports controller we need to order them, so I've moved the order call to there.